Documentation/CRDs/Cluster/external-cluster/topology-for-external-mode.md

```diff
@@ -0,0 +1,121 @@
+# Topology-Based Provisioning
+
+## Scenario
+Applications like Kafka will have a deployment with multiple running instances. Each service instance will create a new claim and is expected to be located in a different zone. Since the application has its own redundant instances, there is no requirement for redundancy at the data layer. A storage class is created that will provision storage from replica 1 Ceph pools that are located in each of the separate zones.
+
+## Configuration Flags
+
+Add the required flags to the script: `create-external-cluster-resources.py`:
+`--topology-pools`: (optional) Comma-separated list of topology-constrained rbd pools
+`--topology-failure-domain-label`: (optional) K8s cluster failure domain label (example: zone, rack, or host) for the topology-pools that match the ceph domain
+ `--topology-failure-domain-values`: (optional) Comma-separated list of the k8s cluster failure domain values corresponding to each of the pools in the `topology-pools` list
+
+The import script will then create a new storage class named `ceph-rbd-topology`.
+
+## Example Configuration
+
+### Ceph cluster
+
+Looking at the osd tree, before jumping to the steps
+```
```

Comment:
Just remove the crush map example. It doesn't show zones anyway, and everyone's CRUSH topology is different. Hopefully the suggested sentence above is sufficient to explain.